lib/graphql/queries.js

```diff
@@ -116,6 +116,12 @@ export const getLoggedInUserQuery = gql`
           balance
           expiryDate
         }
+        payoutMethods {
```

Comment:
@kewitz Sorry for the late comment on this, but I just realized that this goes against https://github.com/opencollective/opencollective/issues/2126. `LoggedInUser` is fetched pretty often, it's a problem that we load this data on every page and not just where it's needed.